java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MBeanable.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2017, Joyent, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.joyent.manta.client;
+
+/**
+ * Interface for objects that can make themselves available as MBeans.
+ */
```

Comment:
Why no `@author` or `@since` tag? Honestly, I never cared for it that much, but one of the early contributors to the project convinced me that it was useful.